### PR TITLE
Add missing commands to the start snippet

### PR DESCRIPTION
**Pull request by patuwwy**
**Review checks:**

These aspects need to be checked by the reviewer:

- [ ] Verify and confirm operation (please post a screenshot) 
- [x] All STH tests pass
- [ ] All [Scramjet Cloud Platform](https://docs.scramjet.org/platform) tests pass
- [x] Documentation is updated or no changes

### CODE REVIEW

**Comment by patuwwy** [comment]
this snippet still needs rework. Why?
```
$ SEQ_ID=$(si seq send ./package.tar.gz)
Request ok: http://127.0.0.1:8000/api/v1/sequence status: 200 OK
$ echo $SEQ_ID
SequenceClient { _id: 'c9ab1abf-5a5e-49b3-9a89-24c35fab06ec', host: HostClient { apiBase: 'http://127.0.0.1:8000/api/v1', client: ClientUtils { apiBase: 'http://127.0.0.1:8000/api/v1', fetch: [Function (anonymous)], normalizeUrlFn: [Function: normalizeUrl], log: [Object] } }, sequenceURL: 'sequence/c9ab1abf-5a5e-49b3-9a89-24c35fab06ec' }
```

default format for cli is pretty https://github.com/scramjetorg/transform-hub/blob/devel/packages/cli/src/lib/config.ts#L149

**Comment by patuwwy** [comment]
Apart from that we can show `si seq deploy` version

File: packages/sth/README.md [hunk 1]
```diff
@@ -77,7 +77,11 @@ Copy a content below and save it as `package.json` file:
 Open a terminal run your program on the hub:
 
 ```bash
-si pack /path/to/my/folder -o ~/package.tar.gz # compress the app to a package
+cd /path/to/my/folder
+
+npm install                                    # install dependencies
+
+si seq pack . -o ~/package.tar.gz              # compress the app to a package
```
Comment (a-tylenda) [inline packages/sth/README.md:84]: These changes must be implemented in `readme.mtpl` file → packages/sth/src/readme.mtpl

**Review by a-tylenda — CHANGES_REQUESTED** [review]
Changes should be added in `readme.mtpl` instead of `readme.md`

**Comment by alicja-gruzdz** [comment]
That's a typical mistake I always make with mtpl :) 
Is there any help still needed here?

**Comment by patuwwy** [comment]
yes, should we `si config set log --format json` to use 

SEQ_ID=$(si seq send ./package.tar.gz | jq .id)

or we can just explain `-`

and do

```
si seq send ./package.tar.gz
si seq start - --args [$APIKEY, "BTC", "EUR"]
si inst stdout -
```
btw what is package.tar.gz ? why do i need api key before first steps?

**Comment by a-tylenda** [comment]
> > yes, should we `si config set log --format json` to use
> > ```
> > SEQ_ID=$(si seq send ./package.tar.gz | jq .id)
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > or we can just explain `-`
> 
> Thanks for explaining. So I see it as a question of what will be easier to use. Also, if we use consistently in the documentations and samples assigning the result of command to a variable then I would change the default CLI to json (omit the additional step to set the formatting to json). Or modify the documentation everywhere accordingly. I think json is such a standard format for developers that nobody will be scared of it when they see it as default. But maybe I underestimate the --pretty cuteness factor :) @a-tylenda what do you think?

I would stick to CLI. I will propose some changes in a commit ✌🏼

**Comment by alicja-gruzdz** [comment]
> I would stick to CLI. I will propose some changes in a commit ✌🏼

excellent, thanks!

**Review by a-tylenda — APPROVED** [review]

**Review by alicja-gruzdz — APPROVED** [review]

File: packages/sth/README.md [hunk 2]
```diff
@@ -74,17 +87,41 @@ Copy a content below and save it as `package.json` file:
...
+
+# alternatively - param will send the last packaged sequence
+si seq start - --args [\"BTC\",\"EUR\"]
+# <instance-id> will be returned
+
+# see the output from the program by reading the stdout stream
+si inst stdout <instance-id>
+
+# alternatively use - param instead of <instance-id>
+si inst stdout -
```
Comment (alicja-gruzdz) [inline packages/sth/README.md:117]: nicely explained

**Pull request merged**